src/components/Login/Login.js

```diff
@@ -80,15 +80,36 @@ class Login extends React.Component {
         <article className="main-panel">
           <header>
             <h1 className="logo-doppler-new">Doppler</h1>
-            <small className="content-signin">
-              {this._('login.you_want_create_account')}{' '}
-              <Link to="/signup" className="link-green uppercase">
-                {this._('login.signup')}
-              </Link>
-            </small>
+            <div className="language-selector">
+              {this.intl.locale === 'es' ? (
+                <div className="option" id="spanish-selector">
+                  <span>
+                    <strong>ES</strong>
+                  </span>
+                  <Link to="/login?lang=en" className="link-green uppercase">
```

Comment:
`link-green` class is still necessary?

```suggestion
                  <Link to="/login?lang=en" className="uppercase">
```

Reply:
fixed

Reply:
Please, test it, I am not sure.

Reply:
yes it's works!